File: python/character_match.py
# Don't forget to run your tests!

def is_character_match(string1, string2):
	string1 = string1.upper()
	string2 = string2.upper()
	arr1 = list(''.join(string1.split(' ')))
	arr1.sort()
	arr2 = list(''.join(string2.split(' ')))
	arr2.sort()
	if len(arr1) != len(arr2):
		return(False)
	for i in range(0, len(arr1)):
		if arr1[i] != arr2[i]:
			return(False)
	return(True)

#print(is_character_match('zach', 'attack'))	

File: js/characterMatch.js
exports.isCharacterMatch = function(string1, string2) {
  // normalize, since we don't care about case	
  string1 = string1.toUpperCase();	
  string2 = string2.toUpperCase();	
  // remove spaces and convert to array	
  let arr1 = string1.split(' ').join('').split('').sort();	
  let arr2 = string2.split(' ').join('').split('').sort();	
  // check to see if same length	
  if (arr1.length !== arr2.length){	
    return false;	
  }	
  //check to see if same values	
  for (let i = 0; i < arr1.length; i++){	
    if (arr1[i] !== arr2[i]){	
      return false;	
    }	
  }	
  return true;
};
